radixdlt/src/integration/java/com/radixdlt/integration/distributed/simulation/application/TimestampChecker.java

```diff
@@ -24,14 +24,18 @@
 import io.reactivex.rxjava3.core.Maybe;
 import io.reactivex.rxjava3.core.Observable;
 
+import java.util.concurrent.TimeUnit;
+
 /**
  * Checks that the first time a ledger update occurs on the network that it is close
  * to the real wall clock time.
  */
 public final class TimestampChecker implements TestInvariant {
+	public static final long ACCEPTABLE_TIME_RANGE = TimeUnit.SECONDS.toMillis(30);
```

Comment:
I think this is OK for now, but I think we need a better invariant checker at some point.

Reply:
Agree.

Reply:
Added ticket for it https://radixdlt.atlassian.net/browse/RPNV1-793